guidelines.md

```diff
@@ -0,0 +1,66 @@
+# Guidelines
+---
+*All references are given for each chapter.*
+
+## Programming Style
+
+[This link](https://google.github.io/styleguide/javaguide.html) provides all information needed.
+
+## Javadoc
+```java
+/**
+ * Multiple lines of Javadoc text are written here,
+ * wrapped normally...
+ */
+public int method(String p1) { ... }
+```
+
+and so
+
+```java
+/*
+ * @(#)Blah.java        1.82 99/03/18
+ *
+ * Copyright (c) 1994-1999 Sun Microsystems, Inc.
+ * 901 San Antonio Road, Palo Alto, California, 94303, U.S.A.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of Sun
+ * Microsystems, Inc. ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Sun.
+ */
+
+
+package java.blah;
```

Comment:
Instead of `blah`, use `foobar` to represent placeholders: https://en.wikipedia.org/wiki/Foobar